### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by danlipsa**

### CODE REVIEW

**Comment by danlipsa** [comment]
Goes with
https://github.com/UV-CDAT/uvcdat-testdata/pull/164

**Comment by danlipsa** [comment]
Fixes:
tests/test_vcs_canvas_background.py
tests/vtk_ui/test_vtk_ui_text_dimensions.py
tests/vtk_ui/test_vtk_ui_widget_show_hide.py

**Comment by danlipsa** [comment]
Still a bunch of tests fail on my machine. I will look at them tomorrow:
	tests/test_vcs_dump_json.py
	tests/test_vcs_draw_logo_on.py
	tests/test_vcs_gms_patterns_hatches.py
	tests/test_vcs_basic_text.py
	tests/test_vcs_iso_celine_part1.py
	tests/test_vcs_custom_logo.py
	tests/test_vcs_update_array.py
	tests/test_vcs_gms_animate_projected_plots.py
	tests/test_vcs_continents.py
	tests/test_vcs_drawLinesAndMarkersLegend.py
	tests/test_vcs_missing_opacity_gms.py
	tests/test_vcs_iso_celine_part2.py
	tests/test_vcs_box_custom_as_def_exts.py
	tests/test_vcs_line_point_extension.py
        tests/test_vcs_hatches_patterns.py

**Comment by danlipsa** [comment]
@doutriaux1 @aashish24 Please review and lets get this merged.

File: tests/vtk_ui/test_vtk_ui_text_dimensions.py [hunk 1]
```diff
@@ -16,19 +16,19 @@ def do(self):
...
                                                text_property, dpi)
-        if w != 299 or h != 24:
+        if w != 298 or h != 23:
             print "couple good descenders width/height changed (%d, %d)"%(w,h)
             return
 
         w, h = vcs.vtk_ui.text.text_dimensions(
               "This one\nis on\nmultiple lines", text_property, dpi)
-        if w != 151 or h != 73:
+        if w != 150 or h != 73:
```
Comment (doutriaux1) [inline tests/vtk_ui/test_vtk_ui_text_dimensions.py]: @danlipsa did you try on both mac and linux, I somehow recall trying this and reverting it for some reason

Comment (danlipsa) [inline tests/vtk_ui/test_vtk_ui_text_dimensions.py]: @doutriaux1 I don't have a mac that is not hires. The hires macs still have the 'odd size' problem, so we cannot run tests there yet. We'll run it there and fix things when the 'odd size' issue get fixed.

**Comment by danlipsa** [comment]
@doutriaux1 Lets merge this. If you work on the failing tests, lets split those so that we don't work on the same thing.
I fixed a few more:

tests/test_vcs_draw_logo_on.py
tests/test_vcs_custom_logo.py
tests/test_vcs_canvas_background.py
tests/vtk_ui/test_vtk_ui_text_dimensions.py
tests/vtk_ui/test_vtk_ui_widget_show_hide.py
and working on: tests/test_vcs_missing_opacity_gms.py

The following are left:
	tests/test_vcs_dump_json.py
	tests/test_vcs_gms_patterns_hatches.py
	tests/test_vcs_basic_text.py
	tests/test_vcs_iso_celine_part1.py
	tests/test_vcs_update_array.py
	tests/test_vcs_gms_animate_projected_plots.py
	tests/test_vcs_continents.py
	tests/test_vcs_drawLinesAndMarkersLegend.py
	tests/test_vcs_iso_celine_part2.py
	tests/test_vcs_box_custom_as_def_exts.py
	tests/test_vcs_line_point_extension.py
        tests/test_vcs_hatches_patterns.py

**Comment by doutriaux1** [comment]
@danlipsa I added a flake8 test. Do you mind rerunning on your computer as well, I'm not 100% sure it's still ok (lots of failiure on my mac, running it on m,y linux as well)

**Comment by danlipsa** [comment]
@doutriaux1 The flake8 test you added passes on my machine.

**Comment by doutriaux1** [comment]
what about the other tests? Just want to be extra safe

**Comment by danlipsa** [comment]
OK I am rerunning everything.

**Comment by aashish24** [comment]
> OK I am rerunning everything.

I am okay with fixing tests in multiple PR's as far as it is passing on @doutriaux1 's and yours machine.

**Comment by danlipsa** [comment]
@doutriaux1 @aashish24 I have the same 13 tests that fail - which I listed earlier. I'll push the fixes which I already have and lets merge this.

**Comment by doutriaux1** [comment]
sounds good!

**Comment by danlipsa** [comment]
@doutriaux1 I had only 3 failures with your new baselines. One of a variable name that was getting a different value than what was in the test - not sure why. I set it so that will have the same value on all machines. Two others where font sizes - It looked like you changed the baselines for mac so I changed the linux baselines. I am ready to merge ...

**Pull request merged**